xwiki-platform-core/xwiki-platform-web/src/main/webapp/resources/uicomponents/suggest/xwiki.selectize.js

```diff
@@ -116,7 +116,7 @@ define('xwiki-selectize', ['jquery', 'selectize', 'xwiki-events-bridge'], functi
       dropdown.addClass('active');
     },
     persist: false,
-    preload: 'focus',
+    preload: true,
```

Comment:
Why is this change needed?

Reply:
That's a separate thing IMO, not related to this issue. And there's no right / perfect choice. Preloading is nice for the user but it's not nice for the server. You don't know if the user will use the picker so you might load the default suggestions for nothing. On the other hand, if we load the default suggestions on focus, as we do now, we don't put the server at work for nothing.

Note that I think @evalica actually "complained" about the fact that you see one suggestion in the list (**the selected value**) and then, after 1 sec, the rest of the suggestions. But that's OK IMO because the selected value you already have it, while the list of suggestions need to be fetched.

In any case, if @evalica really wants to have the suggestions preloaded then she should open a separate issue, which would be an improvement.